Taggers/plugins/DoubleHTagProducer.cc

```diff
@@ -137,12 +142,16 @@ namespace flashgg {
         vetoConeSize_( iConfig.getParameter<double> ( "VetoConeSize" ) ),
         minJetPt_( iConfig.getParameter<double> ( "MinJetPt" ) ),
         maxJetEta_( iConfig.getParameter<double> ( "MaxJetEta" ) ),
+        year_( iConfig.getParameter<unsigned int> ( "year" ) ),
```

Comment:
What is this year variable?? All year dependent config should be done through MetaConditions!

Reply:
This "year" variable is required to calculate MET with xy correction 
(used as input for this function https://github.com/cms-analysis/flashgg/blob/317d5d8a0346c3f545198a5485fa8415edd7d727/Taggers/plugins/DoubleHTagProducer.cc#L906).

 I am reading it from Metacondition itself. You can confirm it from here

1) https://github.com/cms-analysis/flashgg/blob/317d5d8a0346c3f545198a5485fa8415edd7d727/Systematics/python/doubleHCustomize.py#L267
2)https://github.com/cms-analysis/flashgg/blob/317d5d8a0346c3f545198a5485fa8415edd7d727/Taggers/python/flashggDoubleHTag_cfi.py#L54

Kindly let me know if anything look incosistent.